packages/teams-js/src/internal/communication.ts

```diff
@@ -60,10 +60,14 @@ export function initializeCommunication(validMessageOrigins: string[] | undefine
   }
 
   if (!Communication.parentWindow) {
-    GlobalVars.isFramelessWindow = true;
-    /* eslint-disable  @typescript-eslint/ban-ts-comment */
-    // @ts-ignore: window as ExtendedWindow
-    (window as ExtendedWindow).onNativeMessage = handleParentMessage;
+    const extendedWindow = (Communication.currentWindow as unknown) as ExtendedWindow;
```

Comment:
I am not sure if I am stating it right but: Wouldn't this mean that for a user loading/testing an app in a browser which doesnot have any parent or NativeInterface could break their JS unless they use try catch?

Reply:
Correct. Feels like a valid breaking change to take in v2.0. But I'm curious if that's just my opinion and other people see it differently.